Update user-and-permission-management - TO REVIEW.md (#189)
SenseNet/docs.sensenet.com: content/concepts/user-and-permission-management.md

metaTitle: "sensenet - User Management"
metaDescription: "sensenet User Management"
---

sensenet offers a sophisticated user and permission management system, giving you the possibility to apply permission rules according to the logic you prefer.

# Why is it important?

> There are several situations when you have to adjust the rights of the users (when there are some changes in the company, e.g. someone leaves the company), and it is important to make it in a short time. 

Reviewer: ```suggestion
There are several situations when you have to adjust user rights, someone leaves the company for example. In this case it is important to make it in a short time. 
```

# How does it work?

> In Sensenet it is possible to set permissions on role-based or on document level too. There are several built-in goups and users, and also permissions in sensenet, however it is possible to create custom ones as well.

Reviewer: ```suggestion
In sensenet it is possible to set permissions base on roles or on document level too. There are several built-in goups and users, and also permissions in sensenet, however it is possible to create custom ones as well.
```

Author: ```suggestion
In sensenet it is possible to set permissions based on roles or on document level too. There are several built-in goups and users, and also permissions in sensenet, however it is possible to create custom ones as well.
```

> As sensenet is built around the concept “everything is content” user management is very simple, and thanks to the tree structure you can take advantage of the permission inheritance during your work.

Reviewer: ```suggestion
As sensenet is built around the concept “everything is content”, user management is very simple, and thanks to the tree structure you can take advantage of the permission inheritance during your work.
```
